manifests/frpc/templates/deployment.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: frpc
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: frpc
+  template:
+    metadata:
+      labels:
+        app: frpc
+    spec:
+      containers:
+        - image: quay.io/codefresh/frpc:wss1 # temp image until we have an official one
+          name: frpc
+          imagePullPolicy: Always
+          env:
+            - name: AUTHORIZATION
+              valueFrom:
+                secretKeyRef:
+                  name: codefresh-token
+                  key: token
+                  optional: false
+          resources:
+            limits:
+              memory: 100Mi
+            requests:
+              memory: 30Mi
+          volumeMounts:
+            - mountPath: /etc/frp
+              name: frpc-vol
+      volumes:
+        - name: frpc-vol
+          projected:
+            sources:
+              #  - secret:
+              #      name: frpc-certs
+              #      optional: false
+              #      items:
+              #        - key: tls_trusted_ca_file
+              #          path: ca.crt
```

Comment:
what's this? are we not using it?

Reply:
this will be enabled soon, once dev-ops finish setting up the certificate file